applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -41,6 +41,12 @@ The Managed services option is also available if the RHOAS Operator is installed
 ** *Import YAML*: Upload a YAML file to create and define resources for building and deploying applications.
 ** *Upload JAR file*: Upload a JAR file to build and deploy Java applications.
 
+* *Share my Project*: Use this option to add or remove users to a project and provide accessibility options to them.
+
+* *Helm Chart repositories*: Use this option to add Helm Chart repositories in a namespace.
```

Comment:
If you can't change `Helm Chart repositories` because it's an UI element, then don't change it. 

However, since all the other bullet points are actions, if _you_ wrote this, then make it an action, like `Share Helm Chart repositories`

Reply:
@jc-berger - Yeah, as you correctly stated, this is a UI element and hence I am not willing to change it. I will keep it as it is.